maintainer/cuda_build.sh

```diff
@@ -9,11 +9,7 @@ curl "https://api.github.com/repos/espressomd/espresso/statuses/$GIT_COMMIT?acce
       -d "{\"state\": \"$STATUS\", \"context\": \"ICP CUDA build\", \"target_url\": \"$URL\"}"
 
 nvidia-smi
-cd ${CI_PROJECT_DIR}
-cp maintainer/configs/maxset.hpp myconfig.hpp
-mkdir build && cd build
-cmake .. -DCUDA_NVCC_FLAGS="-D_FORCE_INLINES"
-make -j 4 && make check_python
+srcdir="${CI_PROJECT_DIR}" cmake_params='-DCUDA_NVCC_FLAGS="-D_FORCE_INLINES"' with_coverage="true" build_procs=4 maintainer/travis/build_cmake.sh
```

Comment:
I think `-DCUDA_NVCC_FLAGS="-D_FORCE_INLINES` can be removed. We now have that in CMakeLists.txt for NVCC version 8 and GCC version 5.